.travis.yml

```diff
@@ -7,3 +7,13 @@ install:
   - make configure
 script:
   - make test
+
+jobs:
+  include:
+    - stage: deploy
+      python: 3.9
```

Comment:
this promises the deploy step will happen only once and not for every python version.